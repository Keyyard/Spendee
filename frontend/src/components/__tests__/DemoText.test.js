import * as React from 'react';
import renderer from 'react-test-renderer';
import { Text } from 'react-native';

describe('Demo Text Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Text>Hello, Spendee!</Text>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
