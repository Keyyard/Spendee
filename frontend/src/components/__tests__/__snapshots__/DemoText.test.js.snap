// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demo Text Component renders correctly 1`] = `
<Text>
  Hello, Spendee!
</Text>
`;
